ios/BT/Extensions/Foundation/String+Extensions.swift

```diff
@@ -16,6 +16,7 @@ extension String {
   static let bluetoothNotificationIdentifier = "bluetooth-off"
   static let exposureDetectionErrorNotificationTitle = "Error Detecting Exposures"
   static let exposureDetectionErrorNotificationBody = "An error occurred while attempting to detect exposures."
+  static let newExposureNotificationBody = "Someone you were near recently has been diagnosed with COVID-19. Tap for more details."
```

Comment:
Curious - do we have a way to translate the strings that are in native code?

Reply:
@mattThousand line 149 in Exposure managers feeds the string into apple's localization API